crates/bevy_window/src/lib.rs

```diff
@@ -76,6 +77,13 @@ pub struct WindowPlugin {
     /// [`exit_on_all_closed`].
     pub primary_window: Option<Window>,
 
+    /// Settings for the cursor on the primary window.
+    ///
+    /// Defaults to `Some(CursorOptions::default())`.
+    ///
+    /// Has no effect if [`WindowPlugin::primary_window`] is `None`.
+    pub primary_cursor_options: Option<CursorOptions>,
```

Comment:
I left this in for parity with the existing API, but since it's a required component, we could also omit it and let the user override it in an observer for `OnAdd, PrimaryWindow`